### PR TITLE
More detailed setup for blend on other distros

### PR DESCRIPTION
**Pull request by lilkidsuave**
Reworked the entire process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
 
-It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
\ No newline at end of file
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+dependentcies require for both methods
+
+podman 
+
+## Building Manually
```
Comment (kbdharun) [inline README.md:26]:
```suggestion
## Building Manually

```
It is recommended to leave a blank line space after a heading in Markdown.

File: README.md [hunk 2]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
 
 ## Usage
 
-It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
\ No newline at end of file
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+dependentcies require for both methods
+
+podman
```
Comment (kbdharun) [inline README.md:24]:
```suggestion
Both methods require `podman` as a dependency.
```

File: README.md [hunk 3]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
-It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
\ No newline at end of file
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+dependentcies require for both methods
+
+podman 
+
+## Building Manually
+To build manually, you are going to need the following dependentcies
```
Comment (kbdharun) [inline README.md:27]:
```suggestion
To build manually, you are going to need the following dependencies:
```

File: README.md [hunk 4]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
+
+
+
+
+             
+
+
+
+
+
```
Comment (kbdharun) [inline README.md:231]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+dependentcies require for both methods
+
+podman 
+
+## Building Manually
+To build manually, you are going to need the following dependentcies
+
+pacman fakeroot git binutils npm electron(maunaully downloaded from github, see later) bzip2 make gcc
```
Comment (kbdharun) [inline README.md:29]:
```suggestion
`pacman fakeroot git binutils npm electron bzip2 make gcc`
```

File: README.md [hunk 6]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
+
+Install the dependentcies from your package manager except for electron
+
+i.e sudo dnf install pacman fakeroot git binutils npm bzip2 make gcc
```
Comment (kbdharun) [inline README.md:33]:
```suggestion
Install the dependencies from your package manager (except electron) using the following command:

`sudo dnf install pacman fakeroot git binutils npm bzip2 make gcc`
```

File: README.md [hunk 7]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
+
+Clone the repo for building blend
+
+git clone https://github.com/blend-os/blend-pkg
```
Comment (kbdharun) [inline README.md:37]:
```suggestion
Clone the repo for building blend:

`git clone https://github.com/blend-os/blend-pkg`
```

File: README.md [hunk 8]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
+
+Then download electron from https://github.com/electron/electron/releases/download/v24.2.0/electron-v24.2.0-linux-x64.zip
```
Comment (kbdharun) [inline README.md:39]:
```suggestion
Download electron from https://github.com/electron/electron/releases/latest
```
It is recommended to use latest, as the link will get outdated with newer releases.

File: README.md [hunk 9]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
+
+Go back to home
+
+cd
+
+Go to the blend-pkg folder
+
+cd blend-pkg
+
+This is where the guide splits from Arch to non Arch
```
Comment (kbdharun) [inline README.md:87]:
```suggestion

Extract the zip and rename it to electron22

Navigate to Downloads:

`cd Downloads`

Copy it to /usr/lib (on mutable systems):

`sudo cp electron22 -R /usr/lib`

On immutable systems, you will need to edit the pkgbuild in blend-pkg to set a different directory

Go to blend-pkg directory:

`cd blend-pkg`

Edit the file with the editor of choice:

`<text editor> PKGBUILD`

Edit the line to your correct directory:

`electronVer="$(sed s/^v// /usr/lib/electron${_electronversion}/version)"`

i.e on silverblue /var/usrlocal/lib/electron

Navigate back to home:

`cd  && cd Downloads`

Then copy the content to the selected directory

`sudo cp electron22 -R /var/usrlocal`

Navigate back to the blend-pkg folder:

`cd && cd blend-pkg`
```

File: README.md [hunk 10]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
+
+##Arch
```
Comment (kbdharun) [inline README.md:89]:
```suggestion
### Installing blend in Arch
```

File: README.md [hunk 11]
```diff
@@ -17,4 +17,215 @@ Aside from these lines, all the other code in this repository has been written b
...
+
+./blend.sh
+
+Enable the blend-files service
+
+cd /usr/lib/systemd/user
+
+systemctl --user enable blend-files.service
+
+#Done
```
Comment (kbdharun) [inline README.md:121]:
```suggestion
Install `podman`:

`sudo pacman -S podman`

Build the package:

`makepkg -si`

Execute `/etc/profile.d/blend.sh`  to set the path.

Navigate to the following location:

`cd /etc/profile.d/blend.sh` 

Make the script executable:

`chmod +x blend.sh`

Run the script:

`./blend.sh`

Enable the blend-files service:

`cd /usr/lib/systemd/user`

`systemctl --user enable blend-files.service`

```

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution, I have some suggestions for this page.

While, I was in the middle of reviewing I was informed on Discord that they are planning to make the installation a lot easier with a script, so I suggest holding off the PR till it is finalized.

File: README.md (hunk 5 above)
Comment (lilkidsuave) [reply README.md:28]: The PKG build looks specifically for a folder named electron 22, when the current version is 24

**Pull request closed without merging**